.github/workflows/triage.yaml

```
@@ -0,0 +1,52 @@
+on:
+  issues:
+    types:
+    - opened
+    - reopened
+  pull_request:
+    types:
+    - opened
+    - reopened
+jobs:
+  issues:
+    if: github.event_name == 'issues'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: andymckay/labeler@1.0.2
+      with:
+        add-labels: "triage"
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+    - uses: peter-evans/create-or-update-comment@v2
+      if: github.event.action == 'opened'
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        issue-number: ${{ github.event.issue.number }}
+        body: |
+          Thanks for opening this issue :+1:. The team will review it shortly.
+
+          If this is a bug report, make sure to include clear instructions how on to reproduce the problem with [minimal reproducible examples](https://stackoverflow.com/help/minimal-reproducible-example), where possible. If this is a security report, please review our security policy as outlined in [SECURITY.md](https://github.com/kanisterio/kanister/blob/master/SECURITY.md).
+
+          If you haven't already, please take a moment to review our project's [Code of Conduct](https://github.com/kanisterio/kanister/blob/master/CODE_OF_CONDUCT.md) document.
+    - uses: alex-page/github-project-automation-plus@v0.8.1
+      with:
+        repo-token: ${{ secrets.GH_TOKEN }} # must use a PAT here
+        project: Kanister
+        column: To Be Triaged
+  pull_requests:
```

Similar to the comment above

```suggestion
  pull_requests:
    permissions:
      pull-requests: write
```